### DISCUSSION TITLE
iPhone 5c tethered jailbreak/downgrade under linux.

### DISCUSSION THREAD

**Discussion by userse31** [LukeZGD/Legacy-iOS-Kit#329] [General]
This _should_ be possible. The ssh ramdisk works, and as far as I can tell, that environment involves patching signature chacks out of the kernel.

The issue is that I have ZERO clue what I'm doing! But, If I had to guess:

-Boot into ramdisk, sftp everything onto the system partition.
-Reboot the device, pwndfu mode, load patched bootloader/kernel.

The little reading I've done went over my head. I'm pretty much a javascript/c programmer. And a mediocre one at that, lol.

I think the change from the ssh ramdisk would be to mount the root from the system partition on the nand instead of the ram disk.

Any advice would be appreciated.

**Comment by userse31** [comment]
So, it turns out the "just boot" option in the "other utilities" menu is for tethered boot (and not normal, stock booting). It should probably be renamed to "boot tethered" or something.

But, all my work wasn't for nothing! I did at-least _learn_ stuff, and I now know how to upload stuff to pwned dfu mode.

That said, tethered booting still seems to need ironed out.

**Comment by LukeZGD** [comment]
Late reply, but tethered downgrades have been added in v23.11.15

**Reply by userse31** [reply]
I... I need to try this!

**Reply by userse31** [reply]
It works!

![image](https://github.com/LukeZGD/Legacy-iOS-Kit/assets/61165364/081c3747-f38b-44b0-9a1a-a7bd0b94e830)
